core/src/test/java/io/substrait/type/proto/ExchangeRelRoundtripTest.java

```diff
@@ -0,0 +1,108 @@
+package io.substrait.type.proto;
+
+import io.substrait.TestBase;
+import io.substrait.relation.Rel;
+import io.substrait.relation.physical.AbstractExchangeRel;
+import io.substrait.relation.physical.BroadcastExchange;
+import io.substrait.relation.physical.MultiBucketExchange;
+import io.substrait.relation.physical.RoundRobinExchange;
+import io.substrait.relation.physical.ScatterExchange;
+import io.substrait.relation.physical.SingleBucketExchange;
+import io.substrait.relation.physical.TargetType;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class ExchangeRelRoundtripTest extends TestBase {
```

Comment:
minor: this can be package private for JUnit 5
```suggestion
class ExchangeRelRoundtripTest extends TestBase {
```